rules/cloud/azure/azure_privileged_account_sigin_expected_controls.yml

```diff
@@ -0,0 +1,25 @@
+title: Privilege Role Sign-In Outside Expected Controls
+id: cf1e5687-84e1-41af-97a9-158094efef53
+status: experimental
+description: Detects when an account fails a sign-in when it doesn't meet expected controls for admins
+author: Mark Morowczynski '@markmorow', Yochana Henderson, '@Yochana-H'
+date: 2022/08/11
+references:
+    - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-privileged-accounts#things-to-monitor
+logsource:
+    product: azure
+    service: signinlogs
+detection:
+    selection:
+        properties.message|contains|all:
```

Comment:
The value of  `properties.message` is empty (also empty in some other rules)

Reply:
Yes I have several of these last rules I'm unsure on how the best way to handle them. I'm waiting for @Neo23x0 or @frack113 to tell me the best way to handle these. We've submitted a lot of the rules we knew how to do but some of these we werent sure on.

Reply:
Hi like @Neo23x0 said , you can make a PR with many rules. 
Some are some PR with 30 or 50 rules in a time.
For my review , just keep the same topic please.